examples/2_read_serial.py

```diff
@@ -7,11 +7,14 @@
 License: LGPLv3+
 """
 import openPMD
+import os
 
 
 if __name__ == "__main__":
-    series = openPMD.Series("../samples/git-sample/data%T.h5",
-                            openPMD.Access_Type.read_only)
+    series = openPMD.Series(
+        os.path.join("..", "samples", "git-sample", "data%T.h5"),
```

Comment:
ugh, on Windows:
```
RuntimeError: No matching iterations found: ..\samples\git-sample\data%T
```

Reply:
This is very likely related to the ```cwd``` as discussed in https://github.com/openPMD/openPMD-api/pull/207#discussion_r192012409.

Reply:
shouldn't be, it's the same in `2_read_serial.cpp`, no?

Reply:
Indeed that should be the same directory. All the more confusing that [2_read_serial.exe](https://ci.appveyor.com/project/ax3l/openpmd-api/build/1.0.214/job/aqgimr72ysdvrvvx?fullLog=true#L1675) passes as that one uses UNIX-style paths.

Reply:
I will keep it as before for now and we can investigate later in #229